site/src/pages/components/interest-invokers.explainer.mdx

```diff
@@ -4,145 +4,201 @@ name: Interest Invokers (Explainer)
+
+### Other
+
+Th "other" category includes many HID types, including assistive technologies, eye tracking interfaces (e.g. [Apple Vision Pro](https://www.apple.com/apple-vision-pro/)), voice control systems, switch access, and future HIDs that haven't been developed yet.
+
+Since this category of HIDs contains less-commonly-used HID types, and because it also contains future HIDs that don't even exist yet, the specification for `interesttarget` will simply *mandate* that the User Agent provide *a way* for users to both "show interest" and "lose interest" in elements, no matter what HID is in use, without specifying exactly what form that might take. In some cases, this behavior likely cannot be specified, and must be left to the discretion of the user agent. For example, an explicit hand gesture might be needed for the Vision Pro interface. In the to-be-invented case, it stands to reason that devices might introduce new and novel concepts of showing/losing interest.
+
+## Mouse delays
+
+When the HID is a mouse, and hover/de-hover is the method for showing and losing interest, delays are important for a number of reasons:
+
+1. Simply hovering an element should not be enough to show interest in that element, since the user might just be moving the mouse from one place to another on the screen. It would be highly distracting to the user if such a move caused many popovers to show up. For this reason, showing interest needs to be done by hovering an element **for a period of time**.
+
+2. For a similar reason, losing interest cannot be provided by *just* de-hovering the element. For example, in the common case of a link that has a hovercard, it is very common for the hovercard to be separated from the element by some padding space. And it is also common for the user to want to move their mouse *from* the link *to* the hovercard, e.g. to select and copy some text. However, since there's a gap between the elements, moving the mouse across that gap constitutes a de-hover of both the element and its target. For this reason, losing interest needs to be done by de-hovering the element (and its target) **for a period of time**.
+
+Both of these time periods need to be CSS-configurable, because different use cases can require different delays. For example, a responsive gaming site might want very short delays, including potentially zero-delay. On the other hand, information-rich sites such as Wikipedia might want longer delays to ensure their content is easily accessible to all. To achieve this, two properties, `interest-target-show-delay` and `interest-target-hide-delay`, will control the corresponding delays. Because, as described above, both of these delays are fairly critical to proper operation of the interest mechanism, both properties will default to a non-zero value, 0.5 seconds. A shorthand `interest-target-delay` property will control both.
```

Comment:
We probably need further discussion on how these play with different modalities. Like we don't want a delay for touch or keyboard (now we're using a hotkey). But we probably do for say eye tracking, mouse and potentially something like spacial navigation.

Reply:
Hmm. So these delays should really only apply to **mouse** hover delays, and not the other things. For example, the explainer says that keyboard uses a hotkey (no delay needed) and touch uses long-press (I don't think we want to change the long-press delay), then the only thing left for this to apply to would be things in the "other" category. And things in that category are "up to the UA". Perhaps they can just use these delays if it makes sense?

Reply:
Yeah this can perhaps just be solved in the documentation for these rather than in their naming. We'll have bikeshed and what not to narrow this language down anyway.

Reply:
Ack.